checks/evaluation/contributors.go

```diff
@@ -36,32 +39,54 @@ func Contributors(name string,
 ) checker.CheckResult {
 	expectedProbes := []string{
 		contributorsFromOrgOrCompany.Probe,
+		contributorsFromCodeOwners.Probe,
 	}
 
 	if !finding.UniqueProbesEqual(findings, expectedProbes) {
 		e := sce.WithMessage(sce.ErrScorecardInternal, "invalid probe results")
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	numberOfTrue := getNumberOfTrue(findings)
-	reason := fmt.Sprintf("project has %d contributing companies or organizations", numberOfTrue)
+	// only allow max 3 for each
+	numberOfTrueCompanies := int(
+		math.Min(
+			float64(getNumberOfTrue(findings, contributorsFromOrgOrCompany.Probe)),
+			float64(numberCompaniesForTopScore),
+		),
+	)
+	numberOfTrueOwners := int(
+		math.Min(
+			float64(getNumberOfTrue(findings, contributorsFromCodeOwners.Probe)),
+			float64(numberCodeOwnersForTopScore),
+		),
+	)
+	reason := fmt.Sprintf(
+		"project has %d contributing companies or organizations and %d contributing code owners",
+		numberOfTrueCompanies,
+		numberOfTrueOwners,
+	)
 
-	if numberOfTrue > 0 {
+	if numberOfTrueCompanies+numberOfTrueOwners > 0 {
 		logFindings(findings, dl)
 	}
-	if numberOfTrue > numberCompaniesForTopScore {
+	if numberOfTrueCompanies >= numberCompaniesForTopScore && numberOfTrueOwners >= numberCodeOwnersForTopScore {
 		return checker.CreateMaxScoreResult(name, reason)
 	}
 
-	return checker.CreateProportionalScoreResult(name, reason, numberOfTrue, numberCompaniesForTopScore)
+	return checker.CreateProportionalScoreResult(
+		name,
+		reason,
+		numberOfTrueCompanies+numberOfTrueOwners,
+		numberCompaniesForTopScore+numberCodeOwnersForTopScore,
+	)
```

Comment:
i'm hesitant to make half the check something that usually relies on a privileged token